blocks/v2-hero/v2-hero.css

```diff
@@ -0,0 +1,82 @@
+/* stylelint-disable selector-class-pattern */
+.v2-hero {
+  position: relative;
+
+  /* height: 798px; */
+  height: 588px;
+  overflow: hidden;
+}
+
+.v2-hero__image {
+  position: absolute;
+  object-fit: cover;
+  height: 100%;
+  object-position: center;
+  width: 100%;
+}
+
+.v2-hero__content-wrapper {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  justify-content: flex-end;
+}
+
+.v2-hero__content {
+  padding: 40px 16px;
+  display: flex;
+  flex-direction: column;
+  max-width: 1040px;
+}
+
+.v2-hero__heading {
+  margin: 24px 0;
+  color: var(--c-primary-white);
+  font-family: var(--ff-headline-medium);
+  font-size: 28px;
+  font-style: normal;
+  line-height: 120%;
+  letter-spacing: -0.56px;
+  max-width: 600px;
+}
+
+.v2-hero__cta-wrapper {
+  margin: 0;
+}
+
+.v2-hero .v2-hero__cta:any-link {
```

Comment:
This should be a global style for a button.

I’m missing `border-radius: 2px`, and typography is also incorrect. Should be Helvetica instead.

Reply:
I added the `border-radius` for now. We should remove the button styles when we merge the global button styles.